rlclientlib/CMakeLists.txt

```diff
@@ -120,6 +120,7 @@ target_include_directories( rlclientlib
                             ${CMAKE_CURRENT_SOURCE_DIR}
                             ${FLATBUFFERS_INCLUDE_DIRS}
                             ${CMAKE_CURRENT_SOURCE_DIR}/../ext_libs/date
+                            ${CMAKE_CURRENT_SOURCE_DIR}/../ext_libs/vowpal_wabbit/rapidjson/include

```

Comment:
This should not be necessary. Where does rlclientlib require the rapidjson headers?

Reply:
agree it looks like hack.
safe_vw.cc -> parse_example_json.h -> rapidjson